core/server/data/migrations/versions/4.9/05-fix-missed-mobiledoc-url-transforms.js

```diff
@@ -0,0 +1,73 @@
+const logging = require('@tryghost/logging');
+const semver = require('semver');
+const urlUtils = require('../../../../../shared/url-utils');
+const htmlToPlaintext = require('../../../../../shared/html-to-plaintext');
+const mobiledocLib = require('../../../../lib/mobiledoc');
+const {createTransactionalMigration} = require('../../utils');
+
+// in Ghost versions 4.6.1-4.8.4 the 4.0 migration that transfored URLs had a bug
+// that meant urls inside cards in mobiledoc content was not being transformed
+//
+// if the migrations table indicates an upgrade was made from 3.x to 4.6-4.8 then
+// we'll re-run the transforms against post.mobiledoc and re-generate the html
+// and plaintext contents
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const versionUsedFor40Migration = await knex('migrations')
+            .where({
+                name: '18-transform-urls-absolute-to-transform-ready.js',
+                version: '4.0'
+            })
+            .select('currentVersion')
+            .first();
+
+        // migrations table at time of writing only stores major.minor so we need to coerce
```

Comment:
@kevinansfield Is there a particular reason for doing this over `.whereIn('currentVersion', ['4.6', '4.7', '4.8'])` above?

Reply:
Nope, just started off thinking in terms of full versions where server comparison would be useful then continued with that 😬 I'll switch to the simpler comparison!